README.md

```diff
@@ -0,0 +1,61 @@
+# Description
+
+Simple pure python utility for sniffing HTTP/HTTPS decrypted traffic recorded by one of Apple's not-so-well documented
+APIs.
+
+# Installation
+
+```shell
+python3 -m pip install --user -U iosslsniffer
+```
+
+# Prerequisites
+
+## Enable logging global
+
+This package relays on the ability to modify Apples logging global.   **requires a jailbroken device**
+
+### Howto
+
+- Download and install [`rpc_server`](https://github.com/doronz88/rpc-project) on a jailbroken device.
+- Set logging global to `True`
+  - Using integrated `rpc_client`:
+    ```commandline
+    python3 -m iosslsniffer -o out.pcapng -ri RPC_IP -rp RPC_PORT
+    ```
+  - Using `rpc_client` shell
+
+## Apple's CFNetwork profile
+
+This profile is meant for debugging processes using the CFNetwork
```

Comment:
use RPC instead of the profile installation

Reply:
?